src/graph_gui.cpp

```diff
@@ -409,24 +428,36 @@ struct BaseGraphWindow : Window {
 		y = r.top - GetCharacterHeight(FS_SMALL) / 2;
 
 		for (int i = 0; i < (num_hori_lines + 1); i++) {
-			DrawString(r.left - label_width - ScaleGUITrad(4), r.left - ScaleGUITrad(4), y,
-				GetString(STR_GRAPH_Y_LABEL, this->format_str_y_axis, y_label),
-				GRAPH_AXIS_LABEL_COLOUR, SA_RIGHT);
+			if (rtl) {
+				DrawString(r.right + ScaleGUITrad(4), r.right + label_width + ScaleGUITrad(4), y,
+					GetString(STR_GRAPH_Y_LABEL, this->format_str_y_axis, y_label),
+					GRAPH_AXIS_LABEL_COLOUR, SA_LEFT);
```

Comment:
Arabic numbers are right-aligned in all languages.
```suggestion
					GRAPH_AXIS_LABEL_COLOUR, SA_RIGHT);
```

Reply:
Sorry, I forgot that `DrawString` inverts the alignment depending on text direction.
I fixed it myself, and added `SA_FORCE` to prevent the inversion.